shared/data/registry.json

```diff
@@ -1321,7 +1150,7 @@
     "docs": {
       "relationship_name": "vault-kms",
       "service_name": "vault-kms",
-      "url": "/add-services/vault.html"
+      "url": "/add-services/vault-kms.html"
```

Comment:
the file name/location for vault is /add-services/vault.html. See https://docs.upsun.com/add-services/vault.html